### PR TITLE
Sync with current Filebeat aws/cloudtrail

### LINKED ISSUE #79: Update AWS logs package with recent changes in Filebeat
I'm creating this issue to track AWS integration package from recent PRs that made in Filebeat `aws` module:

- [ ] Improve AWS cloudtrail fileset: https://github.com/elastic/beats/pull/18958

- [ ] Fix improper nesting of session_issuer in aws/cloudtrail: https://github.com/elastic/beats/pull/18915

`cloudtrail` data source: https://github.com/elastic/integrations/tree/master/packages/aws/dataset/cloudtrail
cc @leehinman

### PR DESCRIPTION
**Pull request by leehinman**
- session_context fix
- geoip AS for source.ip
- event.category & event.type expansion

Closes #79

### CODE REVIEW

**Comment by mtojek** [comment]
@kaiyan-sheng are there any plans to merge this PR?

**Comment by leehinman** [comment]
> Sorry I missed this PR. Thanks @leehinman for working on it!! Could you do a rebase please? I will test this PR locally.

done.  let me know if you need help testing.

**Comment by kaiyan-sheng** [comment]
I think you need to rerun `packages=aws mage GenerateDocs` to make ci happy 😄

File: packages/aws/dataset/cloudtrail/elasticsearch/ingest-pipeline/default.yml [hunk 1]
```diff
@@ -262,6 +265,344 @@ processors:
...
+          type:
+            - group
+            - info
+        ListAttachedUserPolicies:
+          category:
+            - iam
+          type:
+            - user
+            - info
+        ListGroupsForUser:
```
Comment (kaiyan-sheng) [inline packages/aws/dataset/cloudtrail/elasticsearch/ingest-pipeline/default.yml:454]: This is duplicated with Line 472?

**Pull request merged**